CHANGELOG.md

```diff
@@ -71,6 +71,16 @@ Note that 3.1.0 will be the first API stable release and interfaces in this rele
 
 ## Notable Bug-fixes
 
+### I/O
+
+* The `seqan3::field::cigar` was added to the default fields for reading and writing alignment files.
+  This has the following impact:
+   1. Reading and writing in one line is now possible without additional reference information:
+      `seqan3::alignment_file_output{"foo.sam"} = seqan3::alignment_file_input{"bar.sam"};`
+   2. The `seqan3::alignment_file_output` now accepts `seqan3::field::cigar` and `seqan3::field::alignment`
+      although they store redundant information. For the SAM/BAM format this ambiguity is handled by favoring the CIGAR
+      information at all times if present.
```

Comment:
The commit id is missing: Not sure if the issue id or the PR id or both?

Reply:
Also what does break? Does something break? Usually if I were to expand the default record to a structured binding would be broken. That should be noted.

Reply:
> The commit id is missing: Not sure if the issue id or the PR id or both?

I think we usually used the PR id?

> Also what does break? Does something break? Usually if I were to expand the default record to a structured binding would be broken. That should be noted.

True. Although it is very unlikey that someone used the default fields with structural bindings (binding 15 arguments) I'll add a sentence about that.

Reply:
Is a sentence enough or should I make an example? (This might bloat the changelog too much)

Reply:
I guess since you are right with you assumption I would keep it simple and just write a sentence.